app/ui/expiryOptions.js

```diff
@@ -33,11 +33,7 @@ module.exports = function(state, emit) {
       value => {
         const max = state.user.maxDownloads;
         state.archive.dlimit = Math.min(value, max);
-        if (value > max) {
-          emit('signup-cta', 'count');
-        } else {
-          emit('render');
-        }
+        emit('render');
```

Comment:
We might want to fix this by adjusting the `anon_max_downloads` in config.js instead, so that it's a config change instead of code.  Easier to keep sync'd with upstream.  In fact, I'd say add that change to `config.json` since that leaves the config.js more common to upstream / mainline.

Reply:
good call!